UnityProject/Assets/Scripts/Messages/Server/SetActionUIMessage.cs

```diff
@@ -106,7 +109,10 @@ private void ProcessActionGUI(NetMessage msg)
 			else
 			{
 				// Action pre-placed on a networked object
-				LoadNetworkObject(msg.NetObject);
+				if(!LoadNetworkObject(msg.NetObject))
```

Comment:
dont use ! unless you're inverting
`if(LoadNetworkObject(msg.NetObject) == false)`

Reply:
this is the DNM code for debugging